sympy/printing/pretty/pretty.py

```diff
@@ -1290,6 +1292,10 @@ def _print_GeometryEntity(self, expr):
         # GeometryEntity is based on Tuple but should not print like a Tuple
         return self.emptyPrinter(expr)
 
+    def _print_lerchphi(self, e):
+        func_name = greek_unicode['phi'] if self._use_unicode else 'L'
```

Comment:
This should probably be `greek_unicode['Phi']`. It may also be misleading to write `L` here as it is used for the [Lerch zeta function](https://en.wikipedia.org/wiki/Lerch_zeta_function). It is related to the Lerch transcendent, denoted by capital phi, but not the same.